ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -35,6 +37,7 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
     const {invitationKey, termsOfServiceVersion, onComplete} = this.props;
 
     try {
+      AnalyticsTracker.Registration.DemographicSurvey();
```

Comment:
nit: For separation of responsibilities, I'd probably put this in the `onSubmit` callback below - since the `createAccount` method has no direct relationship to the demographics survey as a step

Reply:
Makes sense. Done.